host-base.json

```diff
@@ -110,7 +110,8 @@
         "sg3_utils",
         "glusterfs-fuse",
         "dnsmasq",
-        "oci-umount"
+        "oci-umount",
+        "origin-node"
```

Comment:
:+1: for adding ``origin-node`` at this stage in development. Reminder to us all that this will be replaced with a node that is built upon upstream merges later.

Reply:
To clarify, `origin-node` was in the main manifest before. This is a small revert of #2, which changed that to `atomic-openshift-node` for RHEL. So we might need to revert this -- the OpenShift repo we're using right now does have it though.